index.js

```diff
@@ -77,30 +73,30 @@ var cssLintPlugin = function(options) {
   });
 };
 
-var defaultReporter = function(file) {
+var defaultReporter = function (file) {
   var errorCount = file.csslint.errorCount;
   var plural = errorCount === 1 ? '' : 's';
 
-  gutil.log(c.cyan(errorCount)+' error'+plural+' found in '+c.magenta(file.path));
+  gutil.log(c.cyan(errorCount) + ' error' + plural + ' found in ' + c.magenta(file.path));
 
-  file.csslint.results.forEach(function(result) {
+  file.csslint.results.forEach(function (result) {
     var message = result.error;
     gutil.log(
       c.red('[') +
       (
-        typeof message.line !== 'undefined' ?
-          c.yellow( 'L' + message.line ) +
-          c.red(':') +
-          c.yellow( 'C' + message.col )
+      typeof message.line !== 'undefined' ?
+        c.yellow('L' + message.line) +
+        c.red(':') +
+        c.yellow('C' + message.col)
```

Comment:
This DOES match my personal coding style -- I don't like to see spaces inside of braces :P